doc/src/fix_wall_gran.rst

```diff
@@ -56,6 +56,8 @@ Syntax
        *shear* values = dim vshear
          dim = *x* or *y* or *z*
          vshear = magnitude of shear velocity (velocity units)
+      *store_contacts* = store contact information for each particle colliding with wall
```

Comment:
this should clarify there is no arg needed, something like
contacts value = none
   store contact info ...